salmon_occurrence/salmon_db.py

```diff
@@ -0,0 +1,77 @@
+from sqlalchemy import Column, Integer, String, Enum, Boolean, Float, ForeignKey, Index, UniqueConstraint
+from sqlalchemy.ext.declarative import declarative_base
+from geoalchemy2 import Geometry
+
+Base = declarative_base()
+
+class Region(Base):
+    __tablename__="region"
+    __table_args__ = {'schema': 'salmon_geometry'}
+    id = Column('region_id', Integer, primary_key=True)
```

Comment:
I frequently wonder whether having columns named differently in the ORM vs. the database is a good idea. Thoughts?

Reply:
I confess I just copied this detail from modelmeta; I don't have an informed opinion. I assume we did it that way for modelemta for _some_ reason!

Reply:
Goodness knows. I find it confusing. It's true that `id` is shorter than `xxxx_id`, but I don't view the savings as very big. I guess so long as non-id columns aren't being renamed, it should be okay. But the latter has always (in modelmeta) seemed arbitrary and unhelpful.